### DISCUSSION TITLE
Welcome to GPUMD Discussions!

### DISCUSSION THREAD

**Discussion by brucefan1983** [brucefan1983/GPUMD#584] [Announcements]
## 👋 Welcome to GPUMD Discussions!
  Here you can start discussions or answer questions related to the GPUMD package, particularly the NEP approach.

**Comment by ytu2023** [comment]
不太会用GitHub呢

**Reply by brucefan1983** [reply]
Well, you have already demonstrated that you can use it.

**Comment by yunzhenlin** [comment]
我想问一下能否用GPU加速一个MentoCarlo模拟粒子体系达到平衡态的问题。我正在做一个有896个粒子的体系，只考虑静电相互作用能。按照c++程序来写的话每一次蒙特卡洛模拟时间消耗就是o(n^2)。我想在计算每个粒子与其它剩余粒子相互作用能的时候用GPU并行来缩减时间。但是我在windows系统下的VS配置好环境后调用核函数并行加速比原来的程序慢了100倍左右。想问一下老师这个加速到底能否成功。

**Comment by brucefan1983** [comment]
Yes you can if you are good at GPU programming, but this question is unrelated to GPUMD.

**Reply by yunzhenlin** [reply]
感谢老师的回答。我正在读老师CUDA编程：基础与实践的书，但是我使用GPU加速程序可以运行，但是却无法成功加速系统。所以我就想来问一下这个GPU加速能否运用于蒙特卡洛模拟的过程。

**Reply by brucefan1983** [reply]
There are many successful examples of accelerating MC simulations using GPUs. So the answer is "yes" for your question.
